File: Conditionals/timeOfDay.py
# Blaine Buckler, Time Of Day Python

from datetime import datetime
import pytz

utah_tz = pytz.timezone("America/Denver") # I couldn't get it accurate to our time zone so I used this index instead. 
utah_time = datetime.now(utah_tz)
print("Local time in Utah:", utah_time.strftime("%H:%M:%S"))
hour = utah_time.hour
print(hour)
if hour > 20 and (hour < 6 or hour < 24):
    print("It is bed time. ")
elif hour == 7: 
    print("Get to school! ")
elif hour < 7 and hour > 16:
    print("It is school time. ")
else: 
    print("it is after school")

File: Conditionals/timeOfDay.c
#include <stdio.h>
#include <stdlib.h>
#include <time.h>

int main(void) {
    setenv("TZ", "America/Denver", 1);
    tzset();

    time_t current = time(NULL);
    struct tm *utah_time = localtime(&current);

    int hour = utah_time->tm_hour;
    
    if (hour > 20 && (hour < 6 || hour < 24)) {
        printf("It is bed time.\n");
    } else if (hour == 7) {
        printf("Get to school!\n");
    } else if (hour >= 7 && hour <=  16) {
        printf("It is school time.\n");
    } else {
        printf("It is after school.\n");
    }
    
    return 0;
}